docs/book/src/reference/metrics.md

```diff
@@ -3,6 +3,53 @@
 By default, controller-runtime builds a global prometheus registry and
 publishes a collection of performance metrics for each controller.
 
+## Exporting Metrics for Prometheus
+
+The project can be enabled to be configured by using `MonitorService` resources from
+the Prometheus Operator. Following the steps to do this setup.
+
+1. Install Prometheus Operator. You can follow up the steps in [Kube Prometheus](https://github.com/coreos/kube-prometheus) project.
```

Comment:
Can we also mention "https://github.com/coreos/prometheus-operator". I personally found this to be simpler than using "kube-promotheus" but I can understand if you want to just stick with the latter.

Reply:
When I was checking it I found problems with `https://github.com/coreos/prometheus-operator` and then, I spoke with their maintainers and they suggested use: https://github.com/coreos/kube-prometheus. Because of this I'd like to not recommend the install/usage of "https://github.com/coreos/prometheus-operator".